cmd/containerboot/kube.go

```diff
@@ -88,9 +75,58 @@ func deleteAuthKey(ctx context.Context, secretName string) error {
 	return nil
 }
 
-var kc *kube.Client
+var kc kube.Client
+
+// setupKube is responsible for doing any necessary configuration and checks to
+// ensure that tailscale state storage and authentication mechanism will work on
+// Kubernetes.
+func (cfg *settings) setupKube(ctx context.Context) error {
+	if cfg.KubeSecret != "" {
```

Comment:
You could save some indentation by inverting this:

```
if cfg.KubeSecret == "" {
    return nil
}
```

Reply:
Thanks ❤️